core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -27,10 +27,10 @@ package org.apache.spark.status.protobuf;
 
 enum JobExecutionStatus {
   JOB_EXECUTION_STATUS_UNSPECIFIED = 0;
-  RUNNING = 1;
-  SUCCEEDED = 2;
-  FAILED = 3;
-  UNKNOWN = 4;
+  JOB_EXECUTION_STATUS_RUNNING = 1;
```

Comment:
This feels redundant to have the name in the enum class and in the constant name?

Reply:
@srowen Yes, but we are following https://developers.google.com/protocol-buffers/docs/style#enums here. The purpose is to avoid naming conflicts. For example, if there is another enum containing `FAILED` or `SUCCEEDED`, the Protobuf compiler won't fail.

Reply:
Oh ok I get it